src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails.jsx

```diff
@@ -3,18 +3,15 @@ import {isEqual} from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import {fetchOrganizationEnvironments} from 'app/actionCreators/environments';
 import {fetchSentryAppComponents} from 'app/actionCreators/sentryAppComponents';
 import {withMeta} from 'app/components/events/meta/metaProxy';
-import {t} from 'app/locale';
 import EventEntries from 'app/components/events/eventEntries';
 import GlobalSelectionStore from 'app/stores/globalSelectionStore';
+import GroupEventDetailsContainer from 'app/views/organizationGroupDetails/groupEventDetailsContainer';
```

Comment:
Hmm, this is a bit weird because it's a circular import... what I might do is make `groupEventDetails` a folder, and have container in `index.jsx` and this file as `groupEventDetails` - that way, whatever imports `groupEventDetails` doesn't need to change and we don't have a circular import (even though webpack handles it fine).